react/features/speaker-stats/components/SpeakerStats.js

```diff
@@ -62,11 +69,15 @@ class SpeakerStats extends Component<Props, State> {
         super(props);
 
         this.state = {
-            stats: this.props.conference.getSpeakerStats()
+            stats: [],
+            criteria: ''
         };
 
+        this.state.stats = this._getSpeakerStats();
```

Comment:
This seems to be called right after `this.state = { stats: [], ... }`, is there a particular reason for not calling it within the state declaration?

Reply:
Yes, that was because _getSpeakerStats uses this.state, but now I fixed the condition with a ? and I placed it correctly.